swap/src/bin/swap.rs

```diff
@@ -99,15 +99,24 @@ async fn main() -> Result<()> {
             let event_loop = tokio::spawn(event_loop.run());
 
             let max_givable = || bitcoin_wallet.max_giveable(TxLock::script_size());
-            let (amount, fees) = determine_btc_to_swap(
+            let (amount, fees) = match determine_btc_to_swap(
                 json,
                 event_loop_handle.request_quote(),
                 bitcoin_wallet.new_address(),
                 || bitcoin_wallet.balance(),
                 max_givable,
                 || bitcoin_wallet.sync(),
             )
-            .await?;
+            .await
+            {
+                Ok(val) => val,
+                Err(error) => match error.downcast::<ZeroQuoteReceived>() {
+                    Ok(_) => {
+                        bail!("The Monero seller you contacted can't trade monero at the moment")
```

Comment:
Small thing but maybe replace this with something like `Seller's XMR balance is currently too low to initiate a swap, please try again later`. We know what caused the error so we can be a little more specific.

Reply:
Much better message hahaha